src/main/io/usb_cdc_hid.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "platform.h"
+
+#ifdef USE_USB_CDC_HID
+
+#include "common/maths.h"
+
+#include "rx/rx.h"
+
+//TODO: Make it platform independent in the future
+#ifdef STM32F4
+#include "vcpf4/usbd_cdc_vcp.h"
+#include "usbd_hid_core.h"
+#elif defined(STM32F7)
+#include "usbd_cdc_interface.h"
+#include "usbd_hid.h"
+#endif
+
+#define USB_CDC_HID_NUM_AXES 8
+
+// In the windows joystick driver, the axes are defined as shown in the third column.
+
+const uint8_t hidChannelMapping[] = {
```

Comment:
https://github.com/betaflight/betaflight/blob/2dd1d742bfb4737b143808cf91181b3498795336/src/main/fc/rc_controls.h#L27-L40
```
ROLL,
PITCH,
AUX3,
THROTTLE,
...

Reply:
What is your point here?

Reply:
After fixing `getMappedChannel`, enum values may be used instead of numbers ...

Reply:
Ah, good point.